src/Draggable/Draggable.js

```diff
@@ -143,8 +157,12 @@ export default class Draggable {
     document.addEventListener('drag:stop', this[onDragStop], true);
     document.addEventListener('drag:pressure', this[onDragPressure], true);
 
-    const defaultPlugins = Object.values(Draggable.Plugins).map((Plugin) => Plugin);
-    const defaultSensors = [MouseSensor, TouchSensor];
+    const defaultPlugins = Object.values(Draggable.Plugins).filter(
+      (Plugin) => !(this.options.exclude.plugins || []).includes(Plugin),
+    );
+    const defaultSensors = Object.values(Draggable.Sensors).filter(
+      (sensor) => !(this.options.exclude.sensors || []).includes(sensor),
+    );
```

Comment:
I think initializing them early makes them more readable. Like https://github.com/zjffun/draggable/commit/b25f818c364f32824818f530dfb0ed59d2a6d9f4

May be it is time to add [@babel/plugin-proposal-optional-chaining · Babel](https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining) 🤔

Reply:
@zjffun , I fixed it based on your changes, but I still hold `exclude` field in `defaultOptions`, just to ensure that we will have a place to see all available options.

https://github.com/bahung1221/draggable/commit/ad65e8e0a27aa9717bf475f2a43718468807ec8d#diff-3fabc15601b4bae571ab86969f145d06R55

Thank you 😄 
But the test was failed, I will check it now, LOL.

Reply:
BTW, do you like `@babel/plugin-proposal-optional-chaining`?
Personal, I avoid adding more packages to project. but I never use it before, so if you think we should use it, I will add it to babel plugins list, don't hesitate 😄

Reply:
I have tried before, it is failed because not the same object.

I’m not sure if it’s good to use the new features, so I select leave it unchanged.

Reply:
I fixed the test, just change it from compare **reference** to compare **value** only.

https://github.com/bahung1221/draggable/commit/3d99e78832a5287acfd33e05cd2fdaf9e37d7a2a#diff-7aa147d42c0b180d8ed3ff8a444159d3R67